src/app/features/home/custom-camera/custom-camera.page.ts

```diff
@@ -142,6 +178,13 @@ export class CustomCameraPage implements OnInit, OnDestroy {
     });
   }
 
+  private removeCurrentCapture() {
+    // TODO: remove file this.curCaptureFilePath to free space
```

Comment:
@sultanmyrza Can we remove the unconfirmed file in this PR, or can you share some details of the difficulty, and we can open a following-up task for it?

Reply:
@bafu, when user take photos or videos it's kept in file system so we need to clean up files. I don't think it's hard by I will add another pull req that will implement this feature (today)